apps/mobile/src/locales/en/messages.po

```diff
@@ -7,42 +7,45 @@ msgstr ""
 "X-Generator: @lingui/cli\n"
 "Language: en\n"
 "Plural-Forms: nplurals=2; plural=(n != 1);\n"
-"X-Crowdin-Project: leather-mobile\n"
-"X-Crowdin-Project-ID: 701559\n"
-"X-Crowdin-Language: en\n"
-"X-Crowdin-File: /src/locales/en/messages.po\n"
-"X-Crowdin-File-ID: 10\n"
 "Project-Id-Version: leather-mobile\n"
 "Language-Team: English\n"
-"PO-Revision-Date: 2025-06-06 10:34\n"
+"PO-Revision-Date: 2025-06-30 14:12\n"
```

Comment:
Working my way through this file I found we have a lot of duplicated text with different IDs